packages/cds-odata-annotation-converter/test/setup.ts

```diff
@@ -68,22 +68,22 @@ export const prepare = async (
     additionalFilesToLoad: string[] = []
 ): Promise<{
     projectRoot: string;
-    cdsArtifacts: CdsArtifactsType;
-    fileCache: Map<string, File>;
+    cdsCompilerFacade: CdsCompilerFacade;
+    metadataElementMap: MetadataElementMap;
 }> => {
     const projectRoot: string = projectRootFolder;
     const roots = await projectAccess.getCdsRoots(projectRoot);
-    const resolvedRoots = cds.resolve(roots ?? []);
-    const cdsFiles = resolvedRoots?.map((uri) => relative(projectRoot, uri));
-    let fileCache: Map<string, File> = new Map();
+    const cdsFiles = (roots ?? [])?.map((uri) => relative(projectRoot, uri));
+    let fileCache: Map<string, string> = new Map();
     if (cdsFiles) {
         const fileList = [...cdsFiles, ...additionalFilesToLoad.filter((f) => !cdsFiles.includes(f))];
         fileCache = (await Promise.all(fileList.map((f) => getFileObj(projectRoot, f)))).reduce((acc, file) => {
-            acc.set(file.fileUri, file);
+            acc.set(file.fileUri, file.fileContent);
             return acc;
-        }, new Map<string, File>());
+        }, new Map<string, string>());
     }
-    const cdsArtifacts = await getCdsArtifacts(projectRoot, cdsServiceName, roots, fileCache);
+    const cdsCompilerFacade = await createCdsCompilerFacadeForRoot(projectRoot, roots, fileCache);
+    const metadataElementMap = cdsCompilerFacade.getMetadata(cdsServiceName);
```

Comment:
We don't need to call it here. it is easier to just return facade and get the metadata if they are required in the test.

Reply:
removed. and simplified the test setup.